README.md

```diff
@@ -41,7 +41,13 @@ the [Google Cloud Console][dev-console]. You can start adding snapshots and log-
 
 ## Running on Google Cloud Platform
 
-Stackdriver debugger agent should work out of the box if your code is running on Google Cloud Platform, just make sure that the [Stackdriver Debugger API][debugger-api] is enabled on your project (this is the default).
+The Stackdriver Debugger Agent should work without manually provided authentication credentials for instances running on Google Cloud Platform, as long as the [Stackdriver Debugger API][debugger-api] access scope is enabled on that instance. For Google App Engine instances, this is automatic if the Debugger API has been enabled for your project (which it is by default).
+
+For new Google Compute Engine instances, you need to explicitly enable the Debugger API access scope. From the web UI, you can do this in one of two ways under **Identity and API access**:
```

Comment:
Ah, I assumed "web UI" to refer to the console as a whole. Maybe specifying the GCE creation page would be sufficient as well.